tests/test-path-manager.c

```diff
@@ -39,15 +40,23 @@ void test_pm_create(void const *test_data)
 
         assert(info->pm         != NULL);
         assert(info->pm->genl   != NULL);
-        assert(info->pm->id     != NULL);
-        assert(info->pm->family != NULL);
         assert(info->pm->nm     != NULL);
+
+        /*
+          Other struct mptcpd_pm fields may not have been initialized
+          yet since they depend on the existence of the "mptcpd"
+          generic netlink family.
+        */
```

Comment:
Is it possible for this test to behave differently based on the timing of mptcpd genl family's appearance? It would be more deterministic to use `l_timeout_create_ms()` to delay the `l_test_run()` call by a given amount of time rather than some number of (probably tight) event loop iterations with `l_idle_create()`.

Reply:
Thanks for pointing out the `l_timeout_create_ms()` function.  I'll give it a try!

Reply:
The two path manager tests now wait a bounded time for the "`mptcp`" generic netlink family to appear before running the tests.  They now fail if they haven't run within the allotted time.  How do they look?